docs/site/src/pages/reference/typescript-client/write-data.mdx

```diff
@@ -124,8 +134,10 @@ try {
 }
 ```
 
-This example will write 100 samples to the `temperature` channel, each roughly
-100ms apart, and will commit all writes when finished.
+This example will write 100 batches of 10 samples to the `temperature` channel, each
+roughly 100ms apart, and will commit all writes when finished. Note that instead of
+passing in series to the frame, you may also pass in singular samples if you wish to
+only write one sample.
```

Comment:
Again, I'm not a fan of starting a sentence with note. If it's an actual note, you could use the `<Note />` component and it will appear in a yellow box. I would either delete this last sentence or show a code snippet where you pass singular samples to the frame.